pinot-core/src/test/java/org/apache/pinot/queries/FluentQueryTest.java

```diff
@@ -101,11 +182,22 @@ public static class DeclaringTable {
       _extraQueryOptions = extraQueryOptions;
     }
 
+    /**
+     * Creates one segment on the first instance with the given content.
+     *
+     * @param content the content of the segment.
+     * @see OnFirstInstance#andSegment(String...) to learn more about the content syntax
+     */
```

Comment:
What are the first and second "instances" referring to exactly? It's a little unclear to me.

Reply:
That is a good question. The framework creates up to 2 servers. Each one is an instance. Most tests won't care much about instances and would be happy using only one, but if you have to test the reduce phase in the broker, it is sometimes useful to receive data from two different instances.

I think that is something I should add in the javadoc

Reply:
Makes sense, thanks!